src/utils/useLocalStorage.ts

```diff
@@ -0,0 +1,40 @@
+import { useState } from 'react';
+
+/** Sync state to local storage so that it persists through a page refresh. */
+export function useLocalStorage<T>(key: string, initialValue: T): [T, ((value: T) => void)] {
```

Comment:
I get it that you copy-pasted this hook from somewhere, but, please, at least make documentation style consistent with the rest of our app

Reply:
I think we should adopt some library with better implementation.